src/status_im2/subs/root.cljs

```diff
@@ -130,115 +130,6 @@
-(reg-root-key-sub :mailserver/mailservers :mailserver/mailservers)
-(reg-root-key-sub :mailserver.edit/mailserver :mailserver.edit/mailserver)
-(reg-root-key-sub :mailserver/state :mailserver/state)
-(reg-root-key-sub :mailserver/pending-requests :mailserver/pending-requests)
-(reg-root-key-sub :mailserver/request-error? :mailserver/request-error)
-(reg-root-key-sub :mailserver/fetching-gaps-in-progress :mailserver/fetching-gaps-in-progress)
-
-;;contacts
-(reg-root-key-sub :contacts/contacts-raw :contacts/contacts)
-(reg-root-key-sub :contacts/current-contact-identity :contacts/identity)
-(reg-root-key-sub :contacts/current-contact-ens-name :contacts/ens-name)
-(reg-root-key-sub :contacts/new-identity :contacts/new-identity)
-(reg-root-key-sub :group/selected-contacts :group/selected-contacts)
-(reg-root-key-sub :contacts/search-query :contacts/search-query)
-
-;;wallet
-(reg-root-key-sub :wallet :wallet)
-(reg-root-key-sub :prices :prices)
-(reg-root-key-sub :prices-loading? :prices-loading?)
-(reg-root-key-sub :wallet.transactions :wallet.transactions)
-(reg-root-key-sub :wallet/custom-token-screen :wallet/custom-token-screen)
-(reg-root-key-sub :wallet/prepare-transaction :wallet/prepare-transaction)
-(reg-root-key-sub :wallet-service/manual-setting :wallet-service/manual-setting)
-(reg-root-key-sub :wallet/recipient :wallet/recipient)
-(reg-root-key-sub :wallet/favourites :wallet/favourites)
-(reg-root-key-sub :wallet/refreshing-history? :wallet/refreshing-history?)
-(reg-root-key-sub :wallet/fetching-error :wallet/fetching-error)
-(reg-root-key-sub :wallet/non-archival-node :wallet/non-archival-node)
-(reg-root-key-sub :wallet/current-base-fee :wallet/current-base-fee)
-(reg-root-key-sub :wallet/slow-base-fee :wallet/slow-base-fee)
-(reg-root-key-sub :wallet/normal-base-fee :wallet/normal-base-fee)
-(reg-root-key-sub :wallet/fast-base-fee :wallet/fast-base-fee)
-(reg-root-key-sub :wallet/current-priority-fee :wallet/current-priority-fee)
-(reg-root-key-sub :wallet/transactions-management-enabled? :wallet/transactions-management-enabled?)
-(reg-root-key-sub :wallet/all-tokens :wallet/all-tokens)
-(reg-root-key-sub :wallet/collectible-collections :wallet/collectible-collections)
-(reg-root-key-sub :wallet/fetching-collection-assets :wallet/fetching-collection-assets)
-(reg-root-key-sub :wallet/collectible-assets :wallet/collectible-assets)
-(reg-root-key-sub :wallet/selected-collectible :wallet/selected-collectible)
-(reg-root-key-sub :wallet/modal-selecting-source-token? :wallet/modal-selecting-source-token?)
-(reg-root-key-sub :wallet/swap-from-token :wallet/swap-from-token)
-(reg-root-key-sub :wallet/swap-to-token :wallet/swap-to-token)
-(reg-root-key-sub :wallet/swap-from-token-amount :wallet/swap-from-token-amount)
-(reg-root-key-sub :wallet/swap-to-token-amount :wallet/swap-to-token-amount)
-(reg-root-key-sub :wallet/swap-advanced-mode? :wallet/swap-advanced-mode?)
-
-;; Wallet 2
```

Comment:
;; Wallet 2
(reg-root-key-sub :wallet-2/scanned-address :wallet-2/scanned-address)

should be restored - @briansztamfater added this but I think the rebase got skewed

Reply:
looks good other than this

Reply:
![](https://github.com/status-im/status-mobile/assets/18485527/e20bbf28-995f-420b-b039-32c87b891098)